### PR TITLE
KRAUS-16: :memo: clean up entry documentation to not be all Java with…

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**
… a callout to python.  Now introduces commons portions and then links to Java and Python content, respectively.

### CODE REVIEW

File: krausening-python/src/krausening/properties/property_encryptor.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class PropertyEncryptor:
 
     Reference: https://resultfor.dev/359470-implement-pbewithhmacsha512andaes-256-of-java-jasypt-in-python.
 
-    See https://bitbucket.org/cpointe/krausening/src/dev/ for details on encrypting values with Jasypt.
+    See https://github.com/TechnologyBrewery/krausening/tree/dev/krausening for details on encrypting values with Jasypt.
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply krausening-python/src/krausening/properties/property_encryptor.py:17]: @sonatype-lift ignore

File: krausening-python/src/krausening/properties/property_manager.py [hunk 2]
```diff
@@ -189,7 +189,7 @@ class EncryptableProperties(Properties):
 
     Reference: https://resultfor.dev/359470-implement-pbewithhmacsha512andaes-256-of-java-jasypt-in-python.
 
-    See https://bitbucket.org/cpointe/krausening/src/dev/ for details on encrypting values with Jasypt.
+    See https:https://github.com/TechnologyBrewery/krausening/tree/dev/krausening for details on encrypting values with Jasypt.
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply krausening-python/src/krausening/properties/property_manager.py:192]: @sonatype-lift ignore

File: krausening-python/README.md [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
+
+Krausening Python is published to PyPI under the [krausening](https://pypi.org/project/krausening/) project and may be installed using any package installer/manager that leverages PyPI.  For example:
 
- Krausening property management and encryption for Python is packaged using the open-source Python Maven plugin [Habushu](https://bitbucket.org/cpointe/habushu) and made available as a [PyPI package](https://pypi.org/project/krausening/).  
+* [Poetry](https://python-poetry.org/) - `poetry add krausening`
+* [pip](https://pip.pypa.io/) - `pip install krausening`
 
 ## Managing Properties with Krausening and Python
 
+**Note: Due to updates the M1 Apple Chip, we strongly recommend using Python >= 3.9 for compatibility reasons.**
```
Comment (ekoniec1) [inline krausening-python/README.md]: A: Technically Poetry enforces this via `python = "^3.9"` - might want to indicate that this is a requirement or just delete

**Review by ekoniec1 — APPROVED** [review]

File: krausening-python/README.md (hunk 3 above)
Comment (d-ryan-ashcraft) [reply krausening-python/README.md]: Good call - updated.

**Pull request merged**